sqlalchemy_ibmi/base.py

```diff
@@ -19,165 +19,206 @@
+                  'ceiling', 'member', 'sqlexception', 'char_length', 'merge',
+                  'sqlstate', 'character_length', 'method', 'sqlwarning',
+                  'clob', 'min', 'sqrt', 'coalesce', 'mod', 'stddev_pop',
+                  'collate', 'module', 'stddev_samp', 'collect', 'multiset',
+                  'submultiset', 'convert', 'national', 'sum', 'corr',
+                  'natural', 'symmetric', 'corresponding', 'nchar',
+                  'tablesample', 'covar_pop', 'nclob', 'timezone_hour',
+                  'covar_samp', 'normalize', 'timezone_minute', 'cube',
+                  'nullif', 'trailing', 'cume_dist', 'numeric', 'translate',
+                  'current_default_transform_group', 'octet_length',
+                  'translation', 'current_role', 'only', 'treat',
+                  'current_transform_group_for_type', 'overlaps', 'true',
+                  'dec', 'overlay', 'uescape', 'decimal', 'percent_rank',
+                  'unknown', 'deref', 'percentile_cont', 'unnest', 'element',
+                  'percentile_disc', 'upper', 'exec', 'power', 'var_pop',
+                  'exp', 'real', 'var_samp', 'false', 'recursive', 'varchar',
+                  'filter', 'ref', 'varying', 'float', 'regr_avgx',
+                  'width_bucket', 'floor', 'regr_avgy', 'window', 'fusion',
+                  'regr_count', 'within', 'asc'}
+
+
+class IBMBoolean(sa_types.Boolean):
+    """IBM i DB2 Boolean class"""
+    def result_processor(self, _, coltype):
         def process(value):
             if value is None:
                 return None
-            else:
-                return bool(value)
+            return bool(value)
+
         return process
 
-    def bind_processor(self, dialect):
+    def bind_processor(self, _):
         def process(value):
             if value is None:
                 return None
-            elif bool(value):
+            if bool(value):
                 return '1'
-            else:
-                return '0'
+            return '0'
+
         return process
 
-class _IBM_Date(sa_types.Date):
 
-    def result_processor(self, dialect, coltype):
+class IBMDate(sa_types.Date):
+    """IBM i DB2 Date class"""
+    def result_processor(self, _, coltype):
         def process(value):
             if value is None:
                 return None
             if isinstance(value, datetime.datetime):
                 value = datetime.date(value.year, value.month, value.day)
             return value
+
         return process
 
-    def bind_processor(self, dialect):
+    def bind_processor(self, _):
         def process(value):
             if value is None:
                 return None
             if isinstance(value, datetime.datetime):
                 value = datetime.date(value.year, value.month, value.day)
             return str(value)
+
         return process
 
+
 class DOUBLE(sa_types.Numeric):
+    """IBM i DB2 Double class"""
```

Comment:
```suggestion
    """Represents a Db2 DOUBLE column"""
```
These could all follow this same pattern